app/(editor)/editor/page.tsx

```diff
@@ -43,7 +43,7 @@ const Page = async ({ searchParams }: PageProps) => {
     notFound();
   }
 
-  const slug = path === 'home' ? '/' : path.replace(/\/$/, '');
+  const slug = path ? path : 'home';
```

Comment:
Updated this so the homepage doesn't error out in the editor - LMK if this breaks anything else